doc/HOWTO_INSTALL_SERVER.txt

```diff
@@ -29,14 +27,14 @@ Installation procedure
 ++++++++++++++++++++++
 
 We recommend that you install this in the home directory of a dedicated
-user, e.g. /home/bibos. In that way, everything can be set up and tested
+user, e.g. /home/os2. In that way, everything can be set up and tested
 without root privileges.
 
 Check out source code, run install scripts, install PostgreSQL DB
 driver:
 
-    cd /home/bibos
-    clone https://github.com/magenta-aps/bibos_admin.git
+    cd /home/os2
+    clone https://github.com/os2borgerpc/admin_site.git
```

Comment:
(Shouldn't this be `git clone`?)

Reply:
Indeed. :slightly_smiling_face:

Reply:
Resolved in commit 0c5a1da417af2234e8b037565f6f1b8a958b8463.